AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -313,15 +315,18 @@ - (NSUInteger)countLogs {
  */
 - (void)migrateDatabase:(void *)db fromVersion:(NSUInteger)version {
   if (version < kMSTargetTokenVersion) {
-    NSString *migrationQuery = [NSString stringWithFormat:@"ALTER TABLE \"%@\" ADD COLUMN \"%@\" "
-                                                           "TEXT",
-                                                          kMSLogTableName, kMSTargetTokenColumnName];
+    NSString *migrationQuery = [NSString
+        stringWithFormat:@"ALTER TABLE \"%@\" ADD COLUMN \"%@\" %@", kMSLogTableName, kMSTargetTokenColumnName, kMSSQLiteTypeText];
     [MSDBStorage executeNonSelectionQuery:migrationQuery inOpenedDatabase:db];
   }
   if (version < kMSTargetKeyVersion) {
-    NSString *migrationQuery = [NSString stringWithFormat:@"ALTER TABLE \"%@\" ADD COLUMN \"%@\" "
-                                                           "TEXT",
-                                                          kMSLogTableName, kMSTargetKeyColumnName];
+    NSString *migrationQuery =
+        [NSString stringWithFormat:@"ALTER TABLE \"%@\" ADD COLUMN \"%@\" %@", kMSLogTableName, kMSTargetKeyColumnName, kMSSQLiteTypeText];
+    [MSDBStorage executeNonSelectionQuery:migrationQuery inOpenedDatabase:db];
+  }
+  if (version < kMSLogPersistencePriorityVersion) {
+    NSString *migrationQuery = [NSString stringWithFormat:@"ALTER TABLE \"%@\" ADD COLUMN \"%@\" %@", kMSLogTableName,
```

Comment:
Should add `DEFAULT 1` at the end of alter table since default value is 0x01 for normal events in CS3 (critical is 0x02).